Makefile

```diff
@@ -22,6 +22,7 @@ build:
 	$(docker-compose) build
 	$(docker-compose) run web rails db:migrate
 	@[ -e tmp/seed ] || make seed
+	$(docker-compose) run web skylight disable_dev_warning
```

Comment:
Since this only needs to be ran once, should this be part of the build command that is meant to be ran many times?

Reply:
Every time I `make build` it needs to be run again, so I think it does need to be here. Let me know if you have a different experience.

Reply:
That might just be my bad.  I thought I remember seeing that warning say that it only needed to be ran once but I was probably misremembering.